src/System.Text.Json/tests/Serialization/TestClasses.SimpleTestClass.cs

```diff
@@ -52,6 +52,7 @@ public class SimpleTestClass : ITestClass
         public ICollection<string> MyStringICollectionT { get; set; }
         public IReadOnlyCollection<string> MyStringIReadOnlyCollectionT { get; set; }
         public IReadOnlyList<string> MyStringIReadOnlyListT { get; set; }
+        public Dictionary<string, string> MyStringToStringDict { get; set; }
```

Comment:
We should definitely add more `Dictionary<string, T>` tests for other Ts (say ints/objects/etc.),but also composing other Ts like collections/dictionary.